{"links": {"self": {"href": "data/repositories/osrf/cloudsim-sim/pullrequests/69/comments/38327535.json"}, "html": {"href": "#!/osrf/cloudsim-sim/pull-requests/69/_/diff#comment-38327535"}}, "deleted": false, "pullrequest": {"type": "pullrequest", "id": 69, "links": {"self": {"href": "data/repositories/osrf/cloudsim-sim/pullrequests/69.json"}, "html": {"href": "#!/osrf/cloudsim-sim/pull-requests/69"}}, "title": "Launch a separate roscore before roslaunching"}, "content": {"raw": "I'm still getting this error on cloudsim. When I tail the rosout.log, there's a bunch of these messages:\n\n~~~\n[ WARN] [1496811723.175096893]: Shutdown request received.\n[ WARN] [1496811723.175141435]: Reason given for shutdown: [new node registered with same name]\n~~~", "markup": "markdown", "html": "<p>I'm still getting this error on cloudsim. When I tail the rosout.log, there's a bunch of these messages:</p>\n<div class=\"codehilite\"><pre><span></span>[ WARN] [1496811723.175096893]: Shutdown request received.\n[ WARN] [1496811723.175141435]: Reason given for shutdown: [new node registered with same name]\n</pre></div>", "type": "rendered"}, "created_on": "2017-06-07T05:03:43.663578+00:00", "user": {"display_name": "Ian Chen", "uuid": "{eaa6fca5-6deb-43f6-907f-971c144735dd}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7Beaa6fca5-6deb-43f6-907f-971c144735dd%7D"}, "html": {"href": "https://bitbucket.org/%7Beaa6fca5-6deb-43f6-907f-971c144735dd%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:10b01d41-a2e9-4a41-a907-e6e2f03b6cd5/1e4adcdf-1946-4280-9aea-eb5f15a7f912/128"}}, "nickname": "Ian Chen", "type": "user", "account_id": "557058:10b01d41-a2e9-4a41-a907-e6e2f03b6cd5"}, "updated_on": "2017-06-07T05:03:43.666726+00:00", "type": "pullrequest_comment", "id": 38327535}